docker/flask/Dockerfile

```diff
@@ -1,9 +1,10 @@
-FROM ubuntu
+FROM ubuntu:17.10
```

Comment:
Why 17.10 and not 18.04? 18.04 is an LTS release, while 17.10 is not.

Reply:
18.04 was just released last week and I would want to fully test it. I didn't realize 17.10 was not lts.